### PR TITLE
fix: use posthog no externals

### PR DESCRIPTION
**Pull request by stephancill**
Disables external imports in posthog 

Fixes https://linear.app/modprotocol/issue/FRA-530/chrome-web-store-app-update

### CODE REVIEW

**Review by michalkvasnicak — APPROVED** [review]

File: apps/extension/background/messages/posthog.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { posthog } from "posthog-js"
+import { posthog } from "posthog-js/dist/module.no-external"
```
Comment (michalkvasnicak) [inline apps/extension/background/messages/posthog.ts:1]: why is this necessary? Just for context

Comment (stephancill) [reply apps/extension/background/messages/posthog.ts:1]: Posthog loads external scripts at runtime which is against Chrome web store policies
